solutions/src/test/java/ru/sberbank/school/task06/CollectionUtilsTest.java

```diff
@@ -0,0 +1,214 @@
+        add(toRemove, tree1);
+        add(toRemove, tree2);
+
+        System.out.println("Before remove\n" + plants);
+
+        removeAll(plants, toRemove);
+
+        System.out.println("To remove\n" + toRemove + "\nAfter remove\n" + plants);
+
+        assertEquals(expectedTarget, plants.get(0));
+    }
+
+    @Test
+    public void containsAllTestTrue() {
+        List<Plant> plants = new ArrayList<>();
+        plants.add(new Tree());
+        plants.add(new Maple());
+        plants.add(new Oak());
+
+        List<Tree> contains = new ArrayList<>();
+        Tree tree1 = (Tree) plants.get(0);
+        Maple tree2 = (Maple) plants.get(1);
+        add(contains, tree1);
+        add(contains, tree2);
+
+        boolean containsAll = containsAll(plants, contains);
+        assertTrue(containsAll);
+    }
+
+    @Test
+    public void containsAllTestFalse() {
+        List<Plant> plants = new ArrayList<>();
+        plants.add(new Tree());
+        plants.add(new Maple());
+        plants.add(new Oak());
+
+        List<Tree> contains = new ArrayList<>();
+        Tree tree1 = (Tree) plants.get(0);
+        Maple tree2 = (Maple) plants.get(1);
+        add(contains, tree1);
+        add(contains, tree2);
+        add(contains, new Oak());
+
+        boolean containsAll = containsAll(plants, contains);
+        assertFalse(containsAll);
+    }
+
+    @Test
+    public void containsAnyTestTrue() {
+        List<Plant> plants = new ArrayList<>();
+        plants.add(new Tree());
+        plants.add(new Maple());
+        plants.add(new Oak());
+
+        List<Tree> contains = new ArrayList<>();
+        Tree tree1 = (Tree) plants.get(0);
+        add(contains, tree1);
+        add(contains, new Maple());
+        add(contains, new Oak());
+
+        boolean containsAny = containsAny(plants, contains);
+        assertTrue(containsAny);
+    }
+
+    @Test
+    public void containsAnyTestFalse() {
+        List<Plant> plants = new ArrayList<>();
+        plants.add(new Tree());
+        plants.add(new Maple());
+        plants.add(new Oak());
+
+        List<Tree> contains = new ArrayList<>();
+        add(contains, new Maple());
+        add(contains, new Oak());
+
+        boolean containsAny = containsAny(plants, contains);
+        assertFalse(containsAny);
+    }
+
+    @Test
+    public void rangeTestComparable() {
+        List<Integer> target = Arrays.asList(3, 4, 5, 6);
+        List<Integer> list = range(Arrays.asList(8, 1, 3, 5, 6, 4), 3, 6);
+        assertThat(list, equalTo(target));
+    }
+
+    @Test
+    public void rangeTestComparator() {
+        Comparator<Plant> comparator = new Comparator<Plant>() {
+            @Override
+            public int compare(Plant o1, Plant o2) {
+                String info1 = o1.getInfo();
+                String info2 = o2.getInfo();
+                if (info1.compareTo(info2) > 0) return 1;
+                if (info1.compareTo(info2) < 0) return -1;
```

Comment:
&#39;if&#39; construct must use &#39;{}&#39;s.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a614402172>*